script/validate-autogen

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I'd rather see this being updated either for `validate-autogen` or all scripts right on this PR. Otherwise, chances are the script will not work until you submitted that follow-up PR. (For instance, on my Mac `/bin/bash` points at the ancient bash 3 version that doesn't work with many features.)